[DOCS] Added description to Audio Filter Effects

henriiquecampos opened the pull request:
Also doccumented Filter member variables and methods.

henriiquecampos committed:
[DOCS] Added description to Audio Filter Effects

Also doccumented Filter member variables and methods.

Noshyaar commented on doc/classes/AudioEffectBandLimitFilter.xml:
```diff
@@ -1,8 +1,10 @@
+		Adds a Band Limit Filter to the Audio Bus.
 	</brief_description>
 	<description>
+		The effect limits the frequences surrounding a range around the [code]Cutoff Hz[/code], and lets only the frequences out of this range pass.
```
frequencies?

henriiquecampos replied:
Also, just so that I don't have an extra request, shall I remove the description for the methods too?

Noshyaar replied:
I don't think there would be any issues leaving it as is. It's up to you.

Still, we need docs team to review this first, to see if the changes comply with guidelines.

cbscribe replied:
The method descriptions are not necessary - they'll go away. However, if you are going to do them (for now, while they are visible), please follow the proper style: "Returns" not "Return", etc.

henriiquecampos replied:
I followed the description of [Sprite](https://github.com/godotengine/godot/blob/master/doc/classes/Sprite.xml#L25) which was approved as
>Return the amount of horizontal frames. See [method set_hframes].

cbscribe replied:
That wasn't approved, it was ignored because that's a getter that's going away in favor of a member variable.

cbscribe commented on doc/classes/AudioEffectBandLimitFilter.xml:
```diff
@@ -1,8 +1,10 @@
+		Adds a Band Limit Filter to the Audio Bus.
 	</brief_description>
 	<description>
+		The effect limits the frequences surrounding a range around the [code]Cutoff Hz[/code], and lets only the frequences out of this range pass.
```
- remove the comma before "and"
- You can remove "The effect" at the beginning. Starting with "Limits the frequencies..." is fine.
- "only the frequencies outside this range"

cbscribe commented on doc/classes/AudioEffectFilter.xml:
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectFilter" inherits="AudioEffect" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Adds a Filter to the Audio Bus.
```
don't capitalize "filter"

cbscribe commented on doc/classes/AudioEffectFilter.xml:
```diff
@@ -1,8 +1,10 @@
+		Adds a Filter to the Audio Bus.
 	</brief_description>
 	<description>
+		The effect lets all frequences other than the [code]Cutoff HZ[/code] pass.
```
Passes all frequencies other than the [member cutoff_hz].

cbscribe commented on doc/classes/AudioEffectFilter.xml:
```diff
@@ -63,17 +72,22 @@
+			Gain amount of the frequences after the filter.
 		</member>
 		<member name="resonance" type="float" setter="set_resonance" getter="get_resonance">
+			Amount of boost in the overtones near the cutoff frequence.
```
Boost amount in the overtones...

cbscribe commented on doc/classes/AudioEffectFilter.xml:
```diff
@@ -63,17 +72,22 @@
+			Threshold frequence for the filter.
 		</member>
 		<member name="dB" type="int" setter="set_db" getter="get_db" enum="AudioEffectFilter.FilterDB">
+			Volume of the effect after the filter in decibels.
```
The effect's volume after...

cbscribe commented on doc/classes/AudioEffectHighPassFilter.xml:
```diff
@@ -1,8 +1,10 @@
+		Adds a High Pass filter to the Audio Bus.
 	</brief_description>
 	<description>
+		The effect cuts frequences lower than the [code]Cutoff HZ[/code], and lets higher frequences pass.
```
- Remove the comma before "and"
- "Cuts frequencies below the [member cutoff_hz]"

henriiquecampos replied:
Shouldn't the Oxford's comma be used, as stated by the [guidelines](http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything) ?

cbscribe replied:
No, that is not an Oxford comma.  "A and B" don't use a comma. "A, B, and C" is when you use the comma.

cbscribe commented on doc/classes/AudioEffectLowPassFilter.xml:
```diff
@@ -1,8 +1,10 @@
+		Adds a Low Pass filter to the Audio Bus.
 	</brief_description>
 	<description>
+		The effect cuts frequences higher than the [code]Cutoff HZ[/code], and lets lower frequences pass.
```
- Remove the comma before "and"
- "Cuts frequencies above the [member cutoff_hz]"

cbscribe commented on doc/classes/AudioEffectHighPassFilter.xml:
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectHighPassFilter" inherits="AudioEffectFilter" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Adds a High Pass filter to the Audio Bus.
```
"High Pass" -> "high pass" or "high-pass", both ways are commonly used.

cbscribe commented on doc/classes/AudioEffectLowPassFilter.xml:
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectLowPassFilter" inherits="AudioEffectFilter" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Adds a Low Pass filter to the Audio Bus.
```
"Low Pass" -> "low pass" or "low-pass"

cbscribe commented on doc/classes/AudioEffectNotchFilter.xml:
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectNotchFilter" inherits="AudioEffectFilter" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Adds a Notch Filter to the Audio Bus.
```
Don't capitalize "Notch Filter"

cbscribe commented on doc/classes/AudioEffectNotchFilter.xml:
```diff
@@ -1,8 +1,10 @@
+		Adds a Notch Filter to the Audio Bus.
 	</brief_description>
 	<description>
+		The effect attenuates the frequences inside a narrow band around the [code]Cutoff HZ[/code], and cuts frequences out of this range.
```
- Remove the comma before "and"
- "Attenuates the frequencies in a narrow band around the [member cutoff_hz]"

cbscribe reviewed (changes requested):
Also, please correct all misspellings of "frequency" and "frequencies"

henriiquecampos committed:
[DOCS] Fixed files based on change requests

Changed capitalization for filter names, removed "The Effect" from descriptions, fixed frequency typos, removed methods descriptions,

henriiquecampos committed:
[DOCS] Fixed Oxford comma standard

Removed commas before enumeration of two items.

cbscribe commented:
Ready to merge. Can you please squash the commits?

akien-mga commented:
Squashed and merged manually as bc667aeada.
